### PR TITLE
[SPARK-7312][SQL] SPARK-6913 broke jdk6 build

### PR DESCRIPTION
**Pull request by yhuai**
JIRA: https://issues.apache.org/jira/browse/SPARK-7312

### CODE REVIEW

**Comment by yhuai** [comment]
@rxin @marmbrus

File: sql/core/src/main/scala/org/apache/spark/sql/jdbc/jdbc.scala [hunk 1]
```diff
@@ -195,7 +195,9 @@ package object jdbc {
 
     override def getMinorVersion: Int = wrapped.getMinorVersion
 
-    override def getParentLogger: java.util.logging.Logger = wrapped.getParentLogger
+    def getParentLogger: java.util.logging.Logger =
```
Comment (srowen) [inline sql/core/src/main/scala/org/apache/spark/sql/jdbc/jdbc.scala]: +1 looks like the right thing to do according to the docs:
http://docs.oracle.com/javase/7/docs/api/java/sql/Driver.html#getParentLogger()

**Comment by rxin** [comment]
LGTM.

**Comment by marmbrus** [comment]
Thanks for the quick fix.  Merged to master.

**Pull request closed without merging**